source/help/getting-started/configuring-notifications.rst

```diff
@@ -63,14 +52,15 @@ Mobile push notifications appear on the lock screen of your mobile device if the
 Browser Tab Notifications
 ----------------------------------------
 
-If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and
-a count of mentions or Direct Messages. Browser tab notifications are available on Firefox and Chrome.
+If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and a count of mentions or Direct Messages. Browser tab notifications are available on Firefox and Chrome.
 
 .. image:: ../../images/browser_notification.png
 
 Muting a Channel
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear as unread unless you are mentioned, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel. 
```

Comment:
Not sure this answers the orignal issue brought up by the github issue, responded there, feel free to make any other changes necessary

https://github.com/mattermost/docs/issues/3187#issuecomment-562241056

Reply:
@justinegeffen see comment above ^. Also you don't need to go to the notification preferences anymoe, mute is in the dropdown menu directly

Reply:
Re-reading this comment, I think it has been addressed by the latest changes. However, if not, let me know and I can rework the update.

Reply:
Following up on this to confirm whether the comment has been addressed or reviewed.